server.py

```
@@ -0,0 +1,255 @@
+    api_key = os.environ.get("GOOGLE_API_KEY")
+    if not api_key:
+        logger.error("GOOGLE_API_KEY not found in environment variables.")
+        await send_to_client(websocket, {"type": "error", "data": {"message": "Server configuration error: Missing API key."}})
+        return
+
+    try:
+        # Configure the client within the handler if preferred, or globally.
+        # Using client = genai.Client() allows for more direct control if needed.
+        # genai.configure(api_key=api_key) # Can be called multiple times, but usually once is enough.
+        # Let's initialize a client instance to be sure.
+        client = genai.GenerativeModel(model_name=MODEL_NAME, tools=[WEATHER_TOOL_SCHEMA])
```

Ruff (0.11.9)

Local variable `client` is assigned to but never used

Remove assignment to unused variable `client`

(F841)